server/api.go

```diff
@@ -29,7 +34,16 @@ func CreateInstallationWebWrapper(i *Installation) (*InstallationWebWrapper, err
 		return nil, err
 	}
 
-	return &InstallationWebWrapper{i, installationLogsURL, provisionerLogsURL}, nil
+	serviceEnvironment := getInstallationServiceEnvironment(i)
+	i.HideSensitiveFields()
+
+	return &InstallationWebWrapper{
+		Installation:        i,
+		CreateAtDate:        cloud.DateStringFromMillis(i.CreateAt),
```

Comment:
This will be a UTC timestamp and if you output it as is it could confuse someone. May be worth converting it to the user's local time? Could either:
1. Leave it as a unix timestamp and allow the front-end to parse it as a date string according to the user's timezone
2. Write the date string in the backend based off the model.User's Timezone field.

Reply:
Yep, agree that supporting timezone conversion would be nice. Going to open up a ticket for a follow-up to improve this.